discordbot/command.py

```diff
@@ -332,8 +332,8 @@ def find_rhino(query: str) -> Card:
     async def notpenny(self, channel: TextChannel, args: str, **_: Dict[str, Any]) -> None:
         """Don't show PD Legality in this channel"""
         existing = configuration.get_list('not_pd')
-        if args == 'server':
-            cid = channel_id(channel)
+        if args == 'server' and channel.guild is not None:
```

Comment:
This will fail in a DM because channel.guild doesn't exist. Needs `getattr(channel, 'guild', None) is not None` I think.